sql/privilege.go

```diff
@@ -40,6 +40,10 @@ func (upl userPrivilegeList) Less(i, j int) bool {
 	return upl[i].User < upl[j].User
 }
 
+func isPrivilegeSet(bits uint32, priv privilege.Kind) bool {
```

Comment:
I would need to put it in privilege/privilege.go and reuse it everywhere in privilege.proto and sql/privilege.go. I'd rather keep it alongside the only code that cares about the bitfield.